free/src/main/scala/cats/free/Coyoneda.scala

```diff
@@ -53,14 +57,20 @@ object Coyoneda {
   /** `F[A]` converts to `Coyoneda[F,A]` for any `F` */
   def lift[F[_], A](fa: F[A]): Coyoneda[F, A] = apply(fa)(identity[A])
 
-  /** Like `lift(fa).map(_k)`. */
-  def apply[F[_], A, B](fa: F[A])(k0: A => B): Aux[F, B, A] =
+  /** Creates a `Coyoneda[F, A]` for any `F`, taking an `F[A]`
+   * and a list of [[Functor.map]]ped functions to apply later
+   */
+  def unsafeApply[F[_], A, B](fa: F[A])(k0: List[Any => Any]): Aux[F, B, A] =
```

Comment:
My bad, I was on my way to making this private. Not because of Any, but because it fixes List.